src/django_grainy/admin.py

```diff
@@ -47,7 +47,8 @@ class GroupPermissionInlineAdmin(admin.TabularInline):
 @admin.register(get_user_model())
 class GrainyUserAdmin(UserAdmin):
     fieldsets = _fieldsets
-    inlines = UserAdmin.inlines + [UserPermissionInlineAdmin]
+    UserAdmin.inlines += (UserPermissionInlineAdmin,)
```

Comment:
i dont think the += on the base class property is needed is it? should be able to just do 

```
inlines = UserAdmin.inlines + (UserPermissionInlineAdmin,)
```
@gablooge

Reply:
@vegu thank you, fixed